code/game/objects/items/ego_weapons/waw.dm

```diff
@@ -15,15 +17,27 @@
 							PRUDENCE_ATTRIBUTE = 60
 							)
 
+/obj/item/ego_weapon/lamp/attack(mob/living/M, mob/living/user)
+	..()
+	for(var/mob/living/L in livinginrange(1, M))
+		force = 25
+		var/userjust = (get_attribute_level(user, JUSTICE_ATTRIBUTE))
+		var/justicemod = 1 + userjust/100
+		force*=justicemod
```

Comment:
This will cause the weapon to have an attack force that has double justice scaling, set the force to 25 AFTER the AoE is done to fix this.

Reply:
Fixed, reset the force at the end.